estate/views/estate_property_offer_views.xml

```diff
@@ -20,15 +20,22 @@
         <field name="name">estate.property.offer.tree</field>
         <field name="model">estate.property.offer</field>
         <field name="arch" type="xml">
-            <tree>
+            <tree editable="bottom" decoration-success="status == 'accepted'" decoration-danger="status == 'refused'">
                 <field name="price"/>
                 <field name="partner_id"/>
                 <field name="validity"/>
                 <field name="date_deadline"/>
-                <button name="action_accept" string="Accept" type="object" icon="fa-check"/>
-                <button name="action_refuse" string="Refuse" type="object" icon="fa-times"/>
-                <field name="status"/>
+                <button name="action_accept" string="Accept" type="object" icon="fa-check" attrs="{'invisible': [('status', 'in', ('accepted', 'refused'))]}"/>
+                <button name="action_refuse" string="Refuse" type="object" icon="fa-times" attrs="{'invisible': [('status', 'in', ('accepted', 'refused'))]}"/>
+                <field name="status" invisible="1"/>
```

Comment:
The chapter says:  You can add the field (property_type_id)  to the list view of offers to make sure it works

Reply:
I read it as "you can add this to verify that it works", but not as a requirement as the offers view from [Chapter 13](https://www.odoo.com/documentation/16.0/developer/tutorials/getting_started/13_inheritance.html#python-inheritance) does not show the property type.